<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>869</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>22</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>数据采集和模型训练测试</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>711</width>
      <height>500</height>
     </rect>
    </property>
    <widget class="QLabel" name="roi_show_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>350</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string>       输入</string>
     </property>
    </widget>
    <widget class="QLabel" name="ROILabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  ROI区域</string>
     </property>
    </widget>
    <widget class="QLabel" name="convas_show_label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>311</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string>       输入</string>
     </property>
    </widget>
    <widget class="QLabel" name="ConvasLabel">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  轮廓图</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>711</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QPushButton" name="CameraButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>打开摄像头</string>
     </property>
    </widget>
    <widget class="QLabel" name="InputEditLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文本输入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="InputEditButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>160</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>自动保存</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="InputEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="ExitButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>0</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QComboBox" name="chooseFunc">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>样本数据集</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>测试数据集</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="InputBoxLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>数字输入</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="InputBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="InputBoxButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>100</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveSingleButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>单张保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SVMTrainButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>230</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>训练</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SVMTestButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>230</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SVMErrorButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>230</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>错误率</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveButton_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>退出保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>加载数据</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>算子维度</string>
     </property>
    </widget>
    <widget class="QComboBox" name="DimsBox">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>64</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>80</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>150</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>     提示信息</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="InfoBrower">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>371</width>
      <height>711</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;这里会显示一些提示信息。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1103</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
